TF_Settings_Web/src/Pages/Visuals/VisualsScreen.tsx

```diff
@@ -92,7 +93,7 @@ const VisualsScreen: React.FC = () => {
     if (!isDesktop() || !hasReadConfig) return <></>;
 
     return (
-        <div className={classes('scroll-div')}>
+        <div className={classes('scroll-div')} onPointerUp={writeConfig}>
```

Comment:
👀 If we are only writing out in this place we could just call `writeVisualsConfig` here (instead of in the reducer) meaning we can change the reducer's signature to just `const reducer = (state: VisualsConfig, content: Partial<VisualsConfig>)` and then simplify the calls to it further (could just call `dispatch({key: value})` and remove updateConfig method)

Reply:
Nice, added this

Reply:
It's still wrapped an extra layer (added seperate comment)